github_pr.py

```diff
@@ -179,19 +184,38 @@ def _check_owner_cannot_ship(owner, comment_users):
 
 
 def _merge_pr(pr, **args):
-    if args['mergecondition_approved_mergers'] or args['mergecondition_approved_mergers_file'] or args['mergecondition_owner_cannot_ship']:
+    if args['condition_approved_mergers'] or args['condition_approved_mergers_file'] or args['condition_owner_cannot_ship']:
         merge_comment_users = [comment.user.login for comment in _load_issue(**args).get_comments().reversed if re.search(".*%s.*" % args['mergecomment'], comment.body)]
-        if args.get('mergecondition_approved_mergers'):
-            merge_comment_users = _check_approved_mergers(args['mergecondition_approved_mergers'], merge_comment_users)
-        if args.get('mergecondition_approved_mergers_file'):
-            merge_comment_users = _check_approved_mergers_file(args['mergecondition_approved_mergers_file'], merge_comment_users)
-        if args.get('mergecondition_owner_cannot_ship'):
+        if args.get('condition_approved_mergers'):
+            merge_comment_users = _check_approved_mergers(args['condition_approved_mergers'], merge_comment_users)
+        if args.get('condition_approved_mergers_file'):
+            merge_comment_users = _check_approved_mergers_file(args['condition_approved_mergers_file'], merge_comment_users)
+        if args.get('condition_owner_cannot_ship'):
             merge_comment_users = _check_owner_cannot_ship(pr.user.login, merge_comment_users)
         if merge_comment_users:
             pr.merge()
     else:
         pr.merge()
 
+def github_check_condition(**args):
+    check_required_fields(['token', 'repo', 'number'], **args)
+    gh = Github(args['token'])
+    repo = gh.get_repo(args['repo'])
+    pr = repo.get_pull(args['number'])
+    issue = _load_issue(**args)
+    last_commit_time = pytz.utc.localize(datetime.strptime(pr.get_commits().reversed[0].commit.raw_data['committer']['date'], '%Y-%m-%dT%H:%M:%SZ')).astimezone(pytz.timezone('US/Eastern'))
```

Comment:
Made this localized now, one concern is, do all of our slaves come up in the "us-east-X" zone? IF a micro slave came online on the west coast it might break this. 

Although im also thinking, do you think we should just keep it in UTC and do the switch to UTC for all times? That way wherever we are we can get the correct checks. (This work can also be done after we get this in place/scale to around the world slaves)